GCDWebServer/Core/GCDWebServerPrivate.h

```diff
@@ -92,6 +92,13 @@
 
 #define __GCDWEBSERVER_LOGGING_FACILITY_COCOALUMBERJACK__
 
+/**
+ * Whether async should be used by log messages, excluding error messages that are always sent sync.
+ **/
+#ifndef LOG_ASYNC_ENABLED
+    #define LOG_ASYNC_ENABLED YES
```

Comment:
Currently I'm using my fork in my projects because I need swift 3...
meanwhile cocoalumberjack did update, so it might be unnecessary anymore... but I had no time to try